### PR TITLE
Add 1.3.10 release note

### PR DESCRIPTION
**Pull request by abbyhu2000**
### Description

Add release note for 1.3.10

### CODE REVIEW

**Review by kavilla — APPROVED** [review]

**Comment by ananzh** [comment]
Could we include https://github.com/opensearch-project/OpenSearch-Dashboards/pull/4045?

File: release-notes/opensearch-dashboards.release-notes-1.3.10.md [hunk 1]
```diff
@@ -0,0 +1,32 @@
...
+
+### 🛡 Security
+
+- [CVE-2020-15366][1.x] Bump ajv from 4.11.8 to 6.12.6 ([#4035](https://github.com/opensearch-project/OpenSearch-Dashboards/pull/4035))
+- [CVE-2022-48285][1.x] Bump jszip from 3.7.1 to 3.10.1 ([#4011](https://github.com/opensearch-project/OpenSearch-Dashboards/pull/4011))
+- [CVE-2021-35065][1.x] Bump glob-parent from 6.0.0 to 6.0.2 ([#4005](https://github.com/opensearch-project/OpenSearch-Dashboards/pull/4005))
+- [CVE-2022-25851][1.x] Bump jpeg-js from 0.4.1 to 0.4.4 ([#3860](https://github.com/opensearch-project/OpenSearch-Dashboards/pull/3860))
+- [CVE-2022-25858][1.x]Bump terser from 4.8.0 to 4.8.1 ([#3786](https://github.com/opensearch-project/OpenSearch-Dashboards/pull/3786))
+- [CVE-2021-23490][1.x] Bump parse-link-header from 1.0.1 to 2.0.0 ([#3820](https://github.com/opensearch-project/OpenSearch-Dashboards/pull/3820))
+- [CVE-2021-3765][1.x] bump validator from 8.2.0 to 13.9.0 ([#3753](https://github.com/opensearch-project/OpenSearch-Dashboards/pull/3753))
```
Comment (ananzh) [inline release-notes/opensearch-dashboards.release-notes-1.3.10.md:11]: bump --> Bump

File: release-notes/opensearch-dashboards.release-notes-1.3.10.md [hunk 2]
```diff
@@ -0,0 +1,32 @@
...
+
+- [TSVB] Fix the link to "serial differencing aggregation" documentation ([#3503](https://github.com/opensearch-project/OpenSearch-Dashboards/pull/3503))
+
+### 📝 Documentation
+
+- Update jest documentation links ([#3939]())
+
+### 🛠 Maintenance
+
+- Adds threshold to code coverage changes for project ([#4050](https://github.com/opensearch-project/OpenSearch-Dashboards/pull/4050))
```
Comment (ananzh) [inline release-notes/opensearch-dashboards.release-notes-1.3.10.md:29]: `Adds` --> `Add`

File: release-notes/opensearch-dashboards.release-notes-1.3.10.md [hunk 3]
```diff
@@ -0,0 +1,32 @@
+# Version 1.3.10 Release Notes
+
+### 🛡 Security
+
+- [CVE-2020-15366][1.x] Bump ajv from 4.11.8 to 6.12.6 ([#4035](https://github.com/opensearch-project/OpenSearch-Dashboards/pull/4035))
+- [CVE-2022-48285][1.x] Bump jszip from 3.7.1 to 3.10.1 ([#4011](https://github.com/opensearch-project/OpenSearch-Dashboards/pull/4011))
+- [CVE-2021-35065][1.x] Bump glob-parent from 6.0.0 to 6.0.2 ([#4005](https://github.com/opensearch-project/OpenSearch-Dashboards/pull/4005))
+- [CVE-2022-25851][1.x] Bump jpeg-js from 0.4.1 to 0.4.4 ([#3860](https://github.com/opensearch-project/OpenSearch-Dashboards/pull/3860))
+- [CVE-2022-25858][1.x]Bump terser from 4.8.0 to 4.8.1 ([#3786](https://github.com/opensearch-project/OpenSearch-Dashboards/pull/3786))
```
Comment (ananzh) [inline release-notes/opensearch-dashboards.release-notes-1.3.10.md:9]: Need a space between [1.x] and Bump

File: release-notes/opensearch-dashboards.release-notes-1.3.10.md [hunk 4]
```diff
@@ -0,0 +1,32 @@
...
+- [CVE-2021-23490][1.x] Bump parse-link-header from 1.0.1 to 2.0.0 ([#3820](https://github.com/opensearch-project/OpenSearch-Dashboards/pull/3820))
+- [CVE-2021-3765][1.x] bump validator from 8.2.0 to 13.9.0 ([#3753](https://github.com/opensearch-project/OpenSearch-Dashboards/pull/3753))
+- [CVE-2022-25758][1.x]Bump scss-tokenizer from 0.3.0 to 0.4.3 ([#3789](https://github.com/opensearch-project/OpenSearch-Dashboards/pull/3789))
```
Comment (ananzh) [inline release-notes/opensearch-dashboards.release-notes-1.3.10.md:12]: Also need a space between 1.x and Bump

**Review by ananzh — CHANGES_REQUESTED** [review]

**Review by kavilla — APPROVED** [review]

**Review by ananzh — APPROVED** [review]

**Review by manasvinibs — APPROVED** [review]

**Pull request merged**